### ISSUE TITLE
Move settings to bottom edge menu

### ISSUE THREAD

**Issue by bittner** [tim-sueberkrueb/ubuntu-hangups#23]
General application settings are in the pull-down menu on top of the screen at the moment. I believe, it could be more pleasant to have all contact and conversation related actions there instead, and move the settings to a separate page that is pulled up from the bottom edge.

The pull-down menu on top of the screen may accommodate settings of the **user account** (not the application behavior). Similar to Telegram, maybe.

The **Settings** page pulled up from the bottom edge may accommodate all settings related to the application behavior, including sort order (see issue #22), cache, etc.

**Comment by timsueberkrueb** [comment]
@bittner thanks for the suggestion. I thought about adding a bottom edge menu as well but for adding new conversations.

**Comment by bittner** [comment]
Isn't the `+` button on top of the screen for new conversations?

So, what are you going to do?

**Comment by timsueberkrueb** [comment]
@bittner yep, it is, but as the bottom edge usually provides a positive/add/compose action (e.g. write a new mail, add a new note, compose a new message) I thought making it adding new conversations will be most intuitive. But I need to think about that, I'm not sure, yet.

**Comment by StuntsPT** [comment]
+1 for settings from bottom edge. But that's my personal preference.

**Comment by zubozrout** [comment]
Hmm, I suppose bottom edge would be much better suited for adding stickers
:). I don't think this is well suited for settings - not to mention Tim
Süberkrüb is right it is not a common place for settings.

**Comment by StuntsPT** [comment]
After a through look at other apps on my Ubuntu phone, I guess you're probably right.
Settings do show up in an icon on top typically. I was thinking about the one exception - the camera app which brings up settings from the bottom edge... Thanks for pointing it out.
